src/sentry/static/sentry/app/stores/configStore.tsx

```diff
@@ -101,4 +101,6 @@ type ConfigStore = Reflux.Store & ConfigStoreInterface;
 
 const ConfigStore = Reflux.createStore(configStoreConfig) as ConfigStore;
 
+window.ConfigStore = ConfigStore;
```

Comment:
Don't do this here, we do this already https://github.com/getsentry/sentry/blob/3f9c71202f641949a120528db4bc60a6ffc9838a/src/sentry/static/sentry/app/bootstrap.tsx#L199